pkg/storage/stores/tsdb/index_client_test.go

```diff
@@ -272,27 +272,55 @@ func TestIndexClient_LabelVolume(t *testing.T) {
 
 	indexClient := NewIndexClient(idx, IndexClientOptions{UseBloomFilters: true})
 
-	t.Run("it returns label volumes from the whole index", func(t *testing.T) {
-		vol, err := indexClient.LabelVolume(context.Background(), "", indexStartYesterday, indexStartToday+1000, 10, nil...)
+	through := indexStartToday + 1000
+	intervals := getIntervalEndpoints(config.ObjectStorageIndexRequiredPeriod, indexStartYesterday.Time(), through.Time(), true)
+
+	t.Run("it returns label volumes from the whole index, separated by ts of interval end", func(t *testing.T) {
```

Comment:
I would still expect volumes to be aggregated by last timestamp or something here. The `interval` here is an internal detail to how this call is made and shouldn't leak in the response type.